lib/puppet/reference/configuration.rb

```diff
@@ -22,10 +22,16 @@
 
     # Now print the data about the item.
     val = object.default
-    if name.to_s == "vardir"
-      val = "/opt/puppetlabs/puppet/cache"
-    elsif name.to_s == "confdir"
-      val = "/etc/puppetlabs/puppet"
+    if name.to_s == 'vardir'
+      val = 'Unix/Linux: /opt/puppetlabs/puppet/cache -- Windows: C:\ProgramData\PuppetLabs\puppet\cache -- Non-root user: ~/.puppetlabs/opt/puppet/cache'
```

Comment:
Technically, it's `%ALLUSERSPROFILE%\Puppet Labs\puppet\cache` on Vista and up, but `%ALLUSERSPROFILE%\Application Data\Puppet Labs\puppet\cache` in 2003, but only 5 more days, so not too worried about that.